### ISSUE TITLE
does not give any result epoch

### ISSUE THREAD

**Issue by humasak** [mhjabreel/STF-RNN#1]
hi,

I tried to run this program. But I do not receive any result. the result only like this:
--
Namespace(batch_size=30, emb_size1=100, emb_size2=6, epochs=100, min_dist=0.2, min_time=1800, rnn_size=20, window_size=2)
Processing user 0
Train on 40 samples, validate on 11 samples
WARNING:tensorflow:From //anaconda3/lib/python3.7/site-packages/tensorflow/python/ops/math_ops.py:3066: to_int32 (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.cast instead.
Epoch 1/100
Processing user 1
--

That's happen for all users. There is no result (accuracy and loss) for each epoch. Would you please help me with this? thank you.

**Comment by archanayadav1408** [comment]
Same problem with me

**Comment by ethanqi1109** [comment]
Same problem with me, do you have any progress?
